core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -104,6 +106,7 @@ open class TileInfo : IsPartOfGameInfoSerialization {
 
     var roadStatus = RoadStatus.None
     var roadIsPillaged = false
+    var roadOwner: CivilizationInfo? = null // either who last built the road or last owner of tile
```

Comment:
Not good. When serializing, we can't include the whole civilization! Instead we need a transient Civilization and a permanent string value of owningCivName

Reply:
Kotlin doesn't seem to have explicit pointer stuff. So is there no smart way for it to store and reference by pointer?

Reply:
Nope, we store the name only and then infer the object when loading the game
Just like with all other objects
There's a lot of interconnectedness in the game and this is a simple way to break it up into bite sized pieces for serialization

Reply:
Ah, that's annoying. K, will check how it's handled elsewhere and copy